frontend/src/metabase/plugins/index.ts

```diff
@@ -152,3 +155,14 @@ export const PLUGIN_APPLICATION_PERMISSIONS = {
     canManageSubscriptions: (_state: any) => true,
   },
 };
+
+export const PLUGIN_GROUP_MANAGERS: PluginGroupManagersType = {
+  UserTypeToggle: () => null as any,
```

Comment:
You could change the type to allow to return `JSX.Element | null` and remove cast to any here